packages/ui-docs-client/src/Select/index.js

```diff
@@ -105,6 +105,7 @@ class Select extends Component {
         renderBeforeInput={renderBeforeInput}
         inputValue={inputValue}
         isShowingOptions={isShowing}
+        onInputChange={() => {}}
```

Comment:
So this is the fix for the error that is thrown now when we load the docs client?

Reply:
Well yes, it is a required prop if value is provided, was missing before too... Couldn't figure out why it didn't throw error before :/